### PR TITLE
Add retry cover for READ handlers; Increase retry delay from 1 second…

### PR DESCRIPTION
**Pull request by boweix-amz**
… to 5 second

*Issue #, if available:*

*Description of changes:*
Change our retry less aggressive.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws-sso-assignment/src/main/java/software/amazon/sso/assignment/CreateHandler.java [hunk 1]
```diff
@@ -78,7 +80,7 @@
                                             throw exception;
                                         }
                                         context.decrementRetryAttempts();
-                                        return ProgressEvent.defaultInProgressHandler(callbackContext, 1, resourceModel);
+                                        return ProgressEvent.defaultInProgressHandler(callbackContext, 5, resourceModel);
```
Comment (andriiglukhyi) [inline aws-sso-assignment/src/main/java/software/amazon/sso/assignment/CreateHandler.java:83]: Non blocking, but like  discussed offline: I would add jitter
Something like 
`BaseHandler:`
```
protected int getRandomDelay() {
    Random rand = new Random(); 
    // Generate random integers in range 0 to 10 
    int rand_int1 = rand.nextInt(10);
}
```
`OtherHandler:`
```
...
return ProgressEvent.defaultInProgressHandler(callbackContext, getRandomDelay(), resourceModel);
...
```

**Review by andriiglukhyi — APPROVED** [review]

**Comment by boweix-amz** [comment]
This CR is large due to the Jacoco coverage requirement. Feel free to skip unit test part that adding all branching coverage.

File: aws-sso-permissionset/aws-sso-permissionset.json [hunk 2]
```diff
@@ -85,7 +85,10 @@
     },
     "InlinePolicy": {
       "description": "The inline policy to put in permission set.",
-      "type": "string",
+      "type": [
+        "object",
+        "string"
+      ],
       "pattern": "[\\w+=,.@-]+",
```
Comment (andriiglukhyi) [inline aws-sso-permissionset/aws-sso-permissionset.json:92]: qq. Does pattern is still applicable property. if type can be either ?

Comment (boweix-amz) [reply aws-sso-permissionset/aws-sso-permissionset.json:92]: Good catch, pattern may not be applied to the combined type. Will remove it and let runtime validation catch it.

**Review by kemmett-amz — APPROVED** [review]

**Pull request merged**